CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing Guidelines
+
+## How to start
+
+1. Create a pull request
+2. Make sure the github action that automatically checks your proofs passes.
+3. Wait for at least one approval from a reviewer
+4. Merge (or poke for a merge, depending on your permissions)
+
+## FAQ
+
+### Where to jump in
+
+1. Find a TODO or Admitted proof and try to prove it.
+2. Read the paper and find something we haven't proved yet.
+3. Open an github issue, if you would like to prove something different that you still feels belong here.
+
+### I have an alternative proof
+
+The more the merrier.
+
+We welcome alternative proofs.  Please submit your proof with a new name, by adding a tick `'` at the end of the function name.
+
+Reason: This is a learning repo and seeing alternatives to the same proof is helpful.  This is not just helpful in pull requests, but also to new comers to the repo and that is why it is still useful to merge them into master.
+
+### Comments
+
+ - Do we like comments? Yes we do.
```

Comment:
Do you have a link handy to a repo with good comment style? I'm still learning the conventions in the Coq community.

Reply:
No I don't, but if you do find one, please share.
At the moment best effort is enough.